src/org/labkey/test/tests/UserPermissionsTest.java

```diff
@@ -267,6 +271,39 @@ public void testAuditLogForGroupUpdates()
         verifyAuditLog("User: " + GAMMA_SUBMITTER_USER + " was deleted from Group: " + GAMMA_SUBMITTER_GROUP_NAME);
     }
 
+    /*
+        Regression coverage for Secure Issue 52229: Current folder filter circumvents permission checking on admin-only tables
+     */
+    @Test
+    public void testFolderFiltersOnAdminOnlyTables()
+    {
+        String listName = TestDataGenerator.randomDomainName("for_Audit_entries");
+
+        goToProjectHome();
+        log("Add list to create the listAuditLogs events");
+        _listHelper.createList(getProjectName(), listName, "List_key");
```

Comment:
Create list via API for efficiency. See usages of `IntListDefinition`.

Reply:
Done